### PR TITLE
[DOC][FQA.md] Correct `transform sql` config

### LINKED ISSUE #4493: [DOC][FQA] Correct `transform sql` config
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

There is an error  in `fqa.md`.It should be to replace "sql" with "query"  on `transform sql` config.

image

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kim-up**
Fix #4493

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [x] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [x] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Review by CheneyYin — COMMENTED** [review]
The case about `join` may be removed, because the join operation was not supported until now.

File: docs/en/faq.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ source {
 
 transform {
 	sql {
-	 sql = """
+	 query = """
```
Comment (TyrantLucifer) [inline docs/en/faq.md]: Now sql plugin does not support join operation, so I think you should change an another example to instead of it.

**Review by CheneyYin — APPROVED** [review]

**Review by TyrantLucifer — APPROVED** [review]
LGTM, thank you for your contribution of Apache SeaTunnel

**Comment by CheneyYin** [comment]
@kim-up ,You may need to invite other members to review as I do not have permission to approve.

**Review by liugddx — APPROVED** [review]

**Pull request merged**